src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -1,6 +1,7 @@
 <Project>
 
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="..\tools\BuildTasks.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
```

Comment:
The use of relative paths in an `Sdk` import probably looks a little weird, but it seems to work beautifully smoothly. The NuGet restore process downloads the SDK package as desired, and the properties are imported into this project as expected.

Avoiding the full `Sdk.props` and `Sdk.targets` is helpful because those files have other consequences. For example, `Sdk.props` indirectly imports `ProjectDefaults.props`, which sets `$(DebugType)`=`embedded` by default.